Makefile

```diff
@@ -361,28 +361,34 @@ localnet-keys:
 	. tests/localosmosis/scripts/add_keys.sh
 
 localnet-build:
-	@DOCKER_BUILDKIT=1 docker-compose -f tests/localosmosis/docker-compose.yml build
-
-localnet-build-state-export:
-	@docker build -t local:osmosis-se --build-arg ID=$(ID) -f tests/localosmosis/mainnet_state/Dockerfile-stateExport .
+	@DOCKER_BUILDKIT=1 COMPOSE_DOCKER_CLI_BUILD=1 docker-compose -f tests/localosmosis/docker-compose.yml build
 
 localnet-start:
 	@docker-compose -f tests/localosmosis/docker-compose.yml up
 
 localnet-startd:
 	@docker-compose -f tests/localosmosis/docker-compose.yml up -d
 
-localnet-start-state-export:
-	@docker-compose -f tests/localosmosis/mainnet_state/docker-compose-state-export.yml up
-
 localnet-stop:
 	@docker-compose -f tests/localosmosis/docker-compose.yml down
 
 localnet-remove: localnet-stop
 	rm -rf $(PWD)/tests/localosmosis/.osmosisd
 
+localnet-build-state-export:
+	@DOCKER_BUILDKIT=1 COMPOSE_DOCKER_CLI_BUILD=1 docker-compose -f tests/localosmosis/state_export/docker-compose.yml build
+
+localnet-start-state-export:
+	@docker-compose -f tests/localosmosis/state_export/docker-compose.yml up
+
+localnet-startd-state-export:
+	@docker-compose -f tests/localosmosis/state_export/docker-compose.yml up -d
+
+localnet-stop-state-export:
+	@docker-compose -f tests/localosmosis/docker-compose.yml down
+
 localnet-remove-state-export:
-	@docker-compose -f tests/localosmosis/mainnet_state/docker-compose-state-export.yml down
+	rm -rf $(PWD)/tests/localosmosis/state_export/.osmosisd
```

Comment:
cc @niccoloraspa @czarcas7ic for these, as they are new or different than the old ones.

Everything else LGTM

Reply:
sure sounds legit